src/Admin/AdminInterface.php

```diff
@@ -339,27 +290,14 @@ public function getFilterParameters(): array;
      */
     public function hasSubject(): bool;
 
-    /**
-     * NEXT_MAJOR: remove this method.
-     *
-     * @param object $object
-     *
-     * @deprecated this feature cannot be stable, use a custom validator,
-     *             the feature will be removed with Symfony 2.2
-     */
-    public function validate(ErrorElement $errorElement, $object): void;
```

Comment:
I don't know about this removal.

The method still exists and is still used in `AbstractAdmin`.

Reply:
Should we undeprecate this then?

Reply:
When looking at this method, I don't think it's useful to keep it.

But, we have at least to
- remove `AbstractAdmin::attachInlineValidator`.
- remove `AbstractAdmin::validate`.
- remove `AbstractAdminExtension::validate`.
- remove `AdminExtensionInterface::validate`.
- update the UPGRADE.md file.

Reply:
I think I will focus on remove all the deprecated things that have a correct path of removal, and keep outside the broken ones.. otherwise this will become impossible to merge

Reply:
Will you revert the change you made for the export too ? 
Then you'll have to add again the translator 😅